.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: go
-go_import_path: github.com/remerge/bolt
+go_import_path: go.etcd.io/bbolt
```

Comment:
So the idea is to have the change in our custom fork as small as possible. Otherwise (like we had it in our sarama) - we will spend too much time rebasing from the upstream.

Also this solution is probably temporary, and changing path to github.com/remerge/bolt makes it less temporary.

Yes, ```replace``` is the price you pay for that, but it worth it imho

Reply:
we have same solution for aerospike now

Reply:
Hmm I don't know, why is it temporary? Do you think this feature will be added in the upstream?
The https://github.com/etcd-io/bbolt#project-status says:
> Bolt is stable, the API is fixed, and the file format is fixed.

I don't think they would add this if just we need it.
Because of this I think it will be pretty much a permanent fork.

Reply:
uff... we need to get rid of permanent forks when possible. I will try to suggest something to boltdb maintainers. If it is will the unaccepted in any way - the smaller we have the patch - the easier to maintain it - so still better not to change path